src/table/components/head/HeadCellMenu.tsx

```diff
@@ -139,8 +141,37 @@ export default function HeadCellMenu({ column, tabIndex }: HeadCellMenuProps) {
                 ],
               },
             ],
+            [
+              {
+                id: 1,
+                itemTitle: translator.get('SNTable.MenuItem.AdjustColumnSize'),
+                onClick: (evt: React.MouseEvent<HTMLLIElement>) => {
+                  evt.stopPropagation();
```

Comment:
I think we can use `preventDefaultBehavior` function for menu items

Reply:
true